src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig

```diff
@@ -31,23 +31,20 @@
     <div class="card">
       <h3 class="card-header">
         <i class="material-icons">filter_none</i> {{ 'Handling'|trans }}
+
+        <span
+          class="help-box"
+          data-container="body"
+          data-toggle="popover"
+          data-trigger="hover"
+          data-placement="right"
+          data-content="{{ 'If you set these parameters to 0, they will be disabled.'|trans({}, 'Admin.Shipping.Help') }} {{ 'Coupons are not taken into account when calculating free shipping.'|trans({}, 'Admin.Shipping.Help') }}"
+          title
```

Comment:
```suggestion
          title=""
```

Reply:
Useless to me, but why not if it seems clearer to you